packages/gatsby-theme-aio/gatsby-config.js

```diff
@@ -14,6 +14,47 @@ require('dotenv').config({
   path: `.env`
 });
 
+if (!process.env.ALGOLIA_APP_ID) {
+  console.error('Missed required parameter: ALGOLIA_APP_ID');
```

Comment:
Not sure if `console.error` is suitable here since algolia is optonal. I'd rather use `console.warn` wdyt ?